.gitignore

```diff
@@ -6,4 +6,5 @@
 /doc/
 /pkg/
 /spec/reports/
+*.sw?
```

Comment:
If you're using vim, I strongly recommend to create `~/.gitignore` and write the below.

```
[._]*.s[a-w][a-z]
[._]s[a-w][a-z]
# session
Session.vim
# temporary
.netrwhist
*~
# auto-generated tag files
tags
```

_(generated by https://www.gitignore.io/api/vim)_

This gitignore affects globally in your machine. You do not have to write this settings at each repository.